docs/cli/regen-api-impl.md

```diff
@@ -0,0 +1,27 @@
+## `ern regen-api-impl`
+
+#### Description
+
+* Regenerate an existing API implementation located in the current working directory.
+
+#### Syntax
+
+`ern regen-api-impl`
+
+**Options**
+
+`<--apiVersion/-v>`
```

Comment:
Please do not use `< >` for options (same applies for `hasConfig` option below).
should rather just be `--apiVersion/-v`
On the command line`< >` indicates mandatory arguments. `[ ]` indicates options but we are not using it in doc